configs/fast_rcnn/README.md

```diff
@@ -1,23 +1,21 @@
 # Fast R-CNN
 
-## Abstract
+> [Fast R-CNN](https://arxiv.org/abs/1504.08083)
 
-<!-- [ABSTRACT] -->
+<!-- [ALGORITHM] -->
+
+## Abstract
 
 This paper proposes a Fast Region-based Convolutional Network method (Fast R-CNN) for object detection. Fast R-CNN builds on previous work to efficiently classify object proposals using deep convolutional networks. Compared to previous work, Fast R-CNN employs several innovations to improve training and testing speed while also increasing detection accuracy. Fast R-CNN trains the very deep VGG16 network 9x faster than R-CNN, is 213x faster at test-time, and achieves a higher mAP on PASCAL VOC 2012. Compared to SPPnet, Fast R-CNN trains VGG16 3x faster, tests 10x faster, and is more accurate.
 
-<!-- [IMAGE] -->
 <div align=center>
 <img src="https://user-images.githubusercontent.com/40661020/143882189-6258c05c-f2a1-4320-9282-7e2f2d502eb2.png"/>
 </div>
 
-<!-- [PAPER_TITLE: Fast R-CNN] -->
-<!-- [PAPER_URL: https://arxiv.org/abs/1504.08083] -->
+## Results and models
```

Comment:
Why did it miss content?

Reply:
The section `Results and Models` in original configs/fast_rcnn/README.md is empty. So I keep this setting.

Reply:
We need to confirm why it is empty. If it is indeed empty, the section should be removed.

Reply:
We do not release the model of Fast RCNN since v2.0. Therefore, we do not have checkpoints for them. I